migration/000001_init.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS Records;
```

Comment:
please enable an option in your editor to add an empty new line at the end of the file